FindNearby/force-app/main/default/lwc/locationsList/locationsList.js

```diff
@@ -17,13 +18,14 @@ export default class LocationsList extends LightningElement {
 
   init = false;
   renderedCallback() {
-    if (!this.init) {
-      this.init = true;
-      this.mainTemplate = this.template.querySelector('.main-container');
-      this.headerElement = this.template.querySelector('.header');
-      this.listElement = this.template.querySelector('.list');
-      this.listElement.style.paddingBottom = this.isIOS ? '34px' : '0';
+    this.setSpinner();
+    this.mainTemplate = this.template.querySelector('.main-container');
+    this.headerElement = this.template.querySelector('.header');
+    this.listElement = this.template.querySelector('.list');
+    if (this.listElement) this.listElement.style.paddingBottom = this.isIos() ? '34px' : '0';
```

Comment:
nit: refactor `34px` as a `const` in your class and then use it here.

Reply:
Done.